src/ios-deploy/ios-deploy.m

```diff
@@ -2087,6 +2087,17 @@ void handle_device(AMDeviceRef device) {
         NSLogOut(@"------ Install phase ------");
         NSLogOut(@"[  0%%] Found %@ connected through %@, beginning install", device_full_name, device_interface_name);
 
+        CFStringRef install_bundle_id = NULL;
+        if (bundle_id != NULL) {
+          install_bundle_id = CFStringCreateWithCString(NULL, bundle_id, kCFStringEncodingUTF8);
+        } else {
+          CFStringRef extracted_bundle_id = copy_bundle_id(url);
```

Comment:
Doesn't this need a CFRelease()?

Reply:
NVM, see below it will be released... have you run the analyzer on this change?

Reply:
Yep did run analyzer, no new issues reported.